Source/GR32_VectorUtils.pas

```diff
@@ -2113,7 +2113,7 @@ function BuildDashedLine(const Points: TArrayOfFloatPoint;
     len1 := Length(Result[0]);
     len2 := Length(Result[J]);
     // Only merge if the first and last points are contributing on a dash
-    if (len1 > 0) and (len2 > 0) and (Result[0][0] = Result[J][len2 - 1]) then
+    if (len1 > 0) and (len2 > 0) and (Result[0][0].X = Result[J][len2 - 1].X) and (Result[0][0].Y = Result[J][len2 - 1].Y) then

```

Comment:
Please change this to:
```
{$IFNDEF FPC}
    if (len1 > 0) and (len2 > 0) and (Result[0][0] = Result[J][len2 - 1]) then
{$ELSE}
    if (len1 > 0) and (len2 > 0) and (Result[0][0].X = Result[J][len2 - 1].X) and (Result[0][0].Y = Result[J][len2 - 1].Y) then
{$ENDIF}
```
otherwise it's bound to be changed back at some point.